### PR TITLE
Replaced requests module with urllib2 for issue #111

### PR DESCRIPTION
**Pull request by giacomocusinato**
Works now fine for me with both Python 2.9 and 3.4

### CODE REVIEW

**Comment by gleitz** [comment]
Thanks for the pull request! Please remove requests from `requirements.txt` if is isn't needed anymore.

**Comment by giacomocusinato** [comment]
This should be it :)

File: howdoi/howdoi.py [hunk 1]
```diff
@@ -13,7 +13,6 @@
 import os
 import random
 import re
-import requests
 import requests_cache
```
Comment (gleitz) [inline howdoi/howdoi.py]: We use requests_cache to speed up query times. Is there a similar library for urllib2?

File: howdoi/howdoi.py [hunk 2]
```diff
@@ -83,9 +85,14 @@ def get_proxies():
...
 def get_result(url):
     try:
-        return requests.get(url, headers={'User-Agent': random.choice(USER_AGENTS)}, proxies=get_proxies()).text
-    except requests.exceptions.SSLError as e:
-        print('[ERROR] Encountered an SSL Error. Try using HTTP instead of '
+        urllib2.install_opener(
+            urllib2.build_opener(
+                urllib2.ProxyHandler(get_proxies())
+            )
+        )
```
Comment (gleitz) [inline howdoi/howdoi.py]: I think the opener only needs to be installed once, correct? Perhaps that can happen at the top of the file.

File: requirements.txt [hunk 3]
```diff
@@ -3,5 +3,4 @@ argparse==1.2.1
 cssselect==0.7.1
 lxml==3.2.4
 pyquery==1.2.6
-requests==2.0.1
 requests-cache==0.4.4
```
Comment (gleitz) [inline requirements.txt]: Let's figure out a replacement for requests-cache

File: howdoi/howdoi.py (hunk 2 above)
Comment (griesmey) [inline howdoi/howdoi.py]: What would be the advantage of using the urllib2 package here?  The requests packages help with readability and is cleaner.

**Comment by gleitz** [comment]
I would rather keep the requests library.

**Pull request closed without merging**